osu.Game.Rulesets.Taiko.Tests/TestSceneLegacyTaikoPlayer.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Database;
+using osu.Game.Rulesets.Taiko.Mods;
+using osu.Game.Skinning;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Rulesets.Taiko.Tests
+{
+    public class TestSceneLegacyTaikoPlayer : PlayerTestScene
+    {
+        protected override Ruleset CreatePlayerRuleset() => new TaikoRuleset();
+
+        [BackgroundDependencyLoader]
+        private void load(SkinManager skins)
+        {
+            skins.CurrentSkinInfo.Value = DefaultLegacySkin.CreateInfo().ToLiveUnmanaged();
```

Comment:
Can this test scene inherit from [`LegacySkinPlayerTestScene`](https://github.com/ppy/osu/blob/master/osu.Game/Tests/Visual/LegacySkinPlayerTestScene.cs) rather than rolling this on its own?

Would also get rid of the ugly `TestSceneTaikoPlayer` workaround, if I'm interpreting what that's there for correctly.

Reply:
Ah, had a feeling a test scene like that existed...